app/models/reservation.rb

```diff
@@ -370,56 +364,53 @@ def update(current_user, new_params, new_notes) # rubocop:disable all
     assign_attributes(new_params)
     changes = self.changes
     new_notes = '' unless new_notes
-    if new_notes.empty? && changes.empty?
-      return self
-    else
-      # write notes header
-      header = "### Edited on #{Time.zone.now.to_s(:long)} by "\
-        "#{current_user.md_link}\n"
-      self.notes = notes ? notes + "\n\n" + header : header
-
-      # add notes if they exist
-      self.notes += "\n\n#### Notes:\n#{new_notes}" unless new_notes.empty?
-
-      # record changes
-      # rubocop:disable BlockNesting
-      unless changes.empty?
-        self.notes += "\n\n#### Changes:"
-        changes.each do |param, diff|
-          case param
-          when 'reserver_id'
-            name = 'Reserver'
-            old_val = diff[0] ? User.find(diff[0]).md_link : 'nil'
-            new_val = diff[1] ? User.find(diff[1]).md_link : 'nil'
-          when 'start_date'
-            name = 'Start Date'
-            old_val = diff[0].to_s(:long)
-            new_val = diff[1].to_s(:long)
-          when 'due_date'
-            name = 'Due Date'
-            old_val = diff[0].to_s(:long)
-            new_val = diff[1].to_s(:long)
-            if checked_out?
-              if overdue? && diff[1] >= Time.zone.today
-                overdue_str = "\nReservation marked as not overdue."
-              elsif !overdue? && diff[1] < Time.zone.today
-                overdue_str = "\nReservation marked as overdue."
-              end
-            end
-          when 'equipment_item_id'
-            name = 'Item'
-            old_val = diff[0] ? EquipmentItem.find(diff[0]).md_link : 'nil'
-            new_val = diff[1] ? EquipmentItem.find(diff[1]).md_link : 'nil'
+    return self if new_notes.empty? && changes.empty?
+    # write notes header
+    header = "### Edited on #{Time.zone.now.to_s(:long)} by "\
+      "#{current_user.md_link}\n"
+    self.notes = notes ? notes + "\n\n" + header : header
```

Comment:
do we prefer this form of concatenation or should this be `"#{notes}\n\n#{header}"`?